integrationTests/package.json

```diff
@@ -21,10 +22,10 @@
     "bundle:install": "npm run bundle:all && npm install",
     "bundle:clean:install": "npm run clean:all && npm run bundle:install",
     "test:integration": "npm run test",
-    "test:integration:redis": "npm run redis && npm run test; npm run redis:shutdown",
```

Comment:
Have you considered adding a step of "npm bundle:tests" or something that just utilizes tsc to transpile the ts caching tests to js for utilization in these tests?  Instead of duplicating functionality?

Reply:
Can u clarify on what is being duplicated?

Reply:
Are these not just javascript file versions of the files in core/tests/*.tests.ts that @bredmond-sf wrote?

Reply:
All caching integration tests are moved to integrationTests package as javascript. There are no more typescript version of these.

Reply:
Okay looks like I was wrong about that. Do you know about that one last integration_test in core and why it remains?

Reply:
I moved it to the tests folder,